docs/quickStartGuide.adoc

```diff
@@ -0,0 +1,84 @@
+== Disctributed Compliance Ledger (DCL)
+
+=== Quick Start Guide for CLI for testnet
+This guide will help you configure DCL `dclcli` and interact with DCL TestNet.
+
+=== Test Net Infrastructure
+TestNet consists of following nodes:<ID>@IP:port are to be used with `dclcli`.
+
+. tcp://54.73.187.91:26657
+. tcp://3.124.76.30:26657
+. tcp://34.224.48.142:26657
+. tcp://44.239.5.82:26657
+
+=== dclcli setup
+These steps will configure `dclcli` on Ubuntu platform.
+
+==== GOLANG 1.3+
+Make sure you have goLang 1.3+ installed.
+https://khongwooilee.medium.com/how-to-update-the-go-version-6065f5c8c3ec
+
+==== Download DCL from repo
```

Comment:
It can be mentioned, that one can use a pre-built version of DCLCLI for Ubuntu: https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/v0.4/dclcli, so no need to clone the repo and run make at all.

Reply:
Added.